Lib/Int.hpp

```diff
@@ -87,221 +74,8 @@ class Int
   static bool stringToFloat(const char*,float& result);
   static bool stringToUnsigned64(const std::string& str,long long unsigned& result);
   static bool stringToUnsigned64(const char* str,long long unsigned& result);
-  static int sign(int i);
-  static int sign(long i);
-  static int sign(double);
-  static int max(int i,int j);
-  static int min(int i,int j);
```

Comment:
`max` and `min` were still used, but as far as I can tell were completely equivalent to `std::max` or `std::min`.